tests/test_api.rs

```diff
@@ -2209,11 +2210,32 @@ fn value_checker() {
     assert!(value == v8::Local::<v8::Promise>::try_from(value).unwrap());
     assert!(value != v8::Object::new(scope));
 
-    let value = eval(scope, context, "new Map()").unwrap();
+    let value = eval(scope, context, "new Map([['r','s'],['v',8]])").unwrap();
     assert!(value.is_map());
     assert!(value == value);
     assert!(value == v8::Local::<v8::Map>::try_from(value).unwrap());
     assert!(value != v8::Object::new(scope));
+    assert_eq!(v8::Local::<v8::Map>::try_from(value).unwrap().size(), 2);
+    let map = v8::Local::<v8::Map>::try_from(value).unwrap();
+    assert_eq!(map.size(), 2);
+    let map_array = map.as_array(scope);
+    assert_eq!(map_array.length(), 4);
+    assert!(
+      map_array.get_index(scope, context, 0).unwrap()
+        == v8::String::new(scope, "r").unwrap()
+    );
+    assert!(
+      map_array.get_index(scope, context, 1).unwrap()
+        == v8::String::new(scope, "s").unwrap()
+    );
+    assert!(
+      map_array.get_index(scope, context, 2).unwrap()
+        == v8::String::new(scope, "v").unwrap()
+    );
+    assert!(
+      map_array.get_index(scope, context, 3).unwrap()
+        == v8::Number::new(scope, 8f64)
+    );
```

Comment:
Please start a new test function for map rather than putting this in the proxy test.

Reply:
Thanks, I add test code and comment

Reply:
I changed it